backend/database/populate.py

```
@@ -0,0 +1,19 @@
+from models.users import User
+from sqlalchemy.orm import Session
+from engine import create_connection
```

Flake8 (7.2.0)

[error] 'engine.create_connection' imported but unused

(F401)